packages/playground-examples/copy/ja/TypeScript/Meta-Types/Indexed Types.ts

```diff
@@ -0,0 +1,37 @@
+// 重複した型に気付く場合があります。
+// よくある例は自動生成されたAPIレスポンスの
+// ネストされたリソースです。
+
+interface ArtworkSearchResponse {
+  artists: {
+    name: string;
+    artworks: {
+      name: string;
+      deathdate: string | null;
+      bio: string;
+    }[];
+  }[];
+}
+
+// もし、このinterfaceが手動で作られていたら、
+// 以下のようにartworksからinterfaceを抜き出してくることはとても簡単です。
+
+interface Artwork {
+  name: string;
+  deathdate: string | null;
+  bio: string;
+}
+
+// しかし、この場合、APIをいじることはできません。
+// 手動でinterfaceをArtworkSearchResponseの
+// 一部であるartworksとして定義することも可能ですが、
+// するとArtwork型はレスポンスが変更されたときに同期されません。
```

Comment:
[IMO]
主語は、 the artwork part of ArtworkSearchResponse と、Artwork が
並列になっているので、そのように訳してみたのですがどうでしょうか？

```suggestion
// そして、もし手動でinterface作った場合に、レスポンスが変わると
// ArtworkSearchResponseのartworksと、
// Artworkの型が不整合を起こす可能性があります。
```